src/components/NewLocationPage/SparkLineBlock/utils.ts

```diff
@@ -50,36 +50,36 @@ interface MetricDescription {
 export const sparkLinesMetricData: {
   [metric in SparkLineMetric]: MetricDescription;
 } = {
-  [SparkLineMetric.CASES]: {
+  [SparkLineMetric.WEEKLY_CASES_PER_100K]: {
     title: 'Cases',
     seriesList: [
       {
-        datasetId: 'rawDailyCases',
+        datasetId: 'weeklyNewCasesPer100k',
       },
       {
-        datasetId: 'smoothedDailyCases',
+        datasetId: 'weeklyNewCasesPer100k',
       },
     ],
   },
-  [SparkLineMetric.DEATHS]: {
-    title: 'Deaths',
+  [SparkLineMetric.ADMISSIONS_PER_100K]: {
+    title: 'Hospitalizations',
```

Comment:
Can you change this (and in the summary stat; sorry I missed that before) to "Admissions" or similar?  I'm not 100% sure what the final copy will be for these, but I'd like to make it clear that we are using admissions data, not the preexisting hospitalizations data.